synapse/handlers/stats.py

```diff
@@ -14,9 +14,15 @@
 # limitations under the License.
 import logging
 from collections import Counter
-from typing import TYPE_CHECKING, Any, Dict, Iterable, Optional, Tuple
-
-from typing_extensions import Counter as CounterType
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Counter as CounterType,
```

Comment:
Looks like we should be able to just use `collections.Counter[str]` in 3.9: https://docs.python.org/3/library/typing.html?highlight=typing%20protocol#typing.Counter